### DISCUSSION TITLE
Where to get my API Key PC_SDK_SUBSCRIPTION_KEY ?

### DISCUSSION THREAD

**Discussion by markmnl** [microsoft/PlanetaryComputer#77] [Q&A] [answered]
I requested access to Planetary Computer preview and got approved.. however the welcome email did not include my PC_SDK_SUBSCRIPTION_KEY

Where can I get this to get higher transfer rates ? (I noticed my downloads slow significantly)

**Comment by mmcfarland** [comment, marked as answer]
Hi @markmnl - you can view and manage your subscription keys at https://planetarycomputer.developer.azure-api.net/. Documentation about how to use then can be found in the [documentation](https://planetarycomputer.microsoft.com/docs/concepts/sas/#supplying-a-subscription-key).

One note is that a subscription key will increase the rate limit on requesting data access tokens ("SAS tokens"), but won't have an effect on actual data transfer speeds, as data is accessed directly from Azure Blog Storage. If you are experiencing slow data access, consider computing in the Azure West Europe datacenter if you are not already (or by using the Planetary Computer Hub, itself operating in West Europe). See ["Computing on the Planetary Computer"](https://planetarycomputer.microsoft.com/docs/concepts/computing/) for more information.

**Reply by markmnl** [reply]
Thanks for where to get the token. Slow transfer speeds were my fault.

**Comment by TomAugspurger** [comment]
You can get one at https://planetarycomputer.developer.azure-api.net/

Can you confirm: were your downloads from Blob Storage slowed, or calls to the SAS API? Can you share a bit about how you’re calling the SAS API (it’s possible to reuse the same token for multiple assets, if you aren’t already).
________________________________
From: Mark Mennell ***@***.***>
Sent: Monday, June 27, 2022 2:51:02 AM
To: microsoft/PlanetaryComputer ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [microsoft/PlanetaryComputer] Where to get my API Key PC_SDK_SUBSCRIPTION_KEY ? (Discussion #77)


I requested access to Planetary Computer preview and got approved.. however the welcome email did not include my PC_SDK_SUBSCRIPTION_KEY

Where can I get this to get higher transfer rates ? (I noticed my downloads slow significantly)

**Comment by pblankenau2** [comment]
When I go to https://planetarycomputer.developer.azure-api.net/ I no longer see any way of accessing the PC_SDK_SUBSCRIPTION_KEY.  Are those keys only available for folks paying for planetary computer pro now?

**Reply by gatienc** [reply]
Having a similar issue so I'm allowing myself to send a heads up.

**Reply by BlackLion201** [reply]
I'm also not seeing a way of obtaining it.
